.rubocop.yml

```diff
@@ -512,3 +512,21 @@ Style/Encoding:
 # so once RE-5565 is resolved, this should be enabled.
 Metrics/PerceivedComplexity:
   Enabled: true
+
+# This defaults to a much smaller number, but correcting the offending methods
+# will take a bit of work, so let's just not write anything more complex than
+# what we already have!
+Metrics/AbcSize:
+  Max: 45
```

Comment:
Consider instead leaving this set to the default, letting it flag the offending methods, and then using inline overrides to note them in comments:

```
def someinsanemethod # rubocop:disable Metrics/AbcSize
```

(I think this will work; untested but I'll test it today)